### ISSUE TITLE
UI: Create PriceList form

### ISSUE THREAD

**Issue by kasperkristensen** [medusajs/admin#513]
**What**

We want to create a form to handle creating new Price Lists. 

See [figma](https://www.figma.com/file/a5Mb7n6lelmmFU28wqQybY/Customer-Groups%2C-Promotions-%26-Pricing?node-id=570%3A11647)

**Note**

You should skip implementing the `Prices` section for now, as the necessary component will be part of a separate PR.

Depends on #512
